setup.py

```diff
@@ -1,10 +1,16 @@
+from pathlib import Path
 from setuptools import setup, Extension
 
+with Path(__file__).parent.joinpath('README.md').open() as f:
```

Comment:
Pourquoi pas with `open("README.md")`?

Reply:
No good reason, just a remanent neuron shooting in the dark of my mind "absolute path! absolute path!". ;)
Fixed!

Reply:
Side note: that may also have been a good candidate for using https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text for a one liner:

```
long_description = Path('README.md').read_text()
```

Thoughts?

Reply:
Not only saves a line, saves two, by putting it line 10 instead of in a temporary variable, so I can't say I don't like it \o/

Only valid for projects that are not keeping Python<3.5 compatibility though.

Reply:
> Only valid for projects that are not keeping Python<3.5 compatibility though.

I thought this was the case here, but the setup.py only says python 3 (Travis only build on 3.5 BTW).

@1st1 is httptools supposed to work with 3.4?
Also, what about adding python 3.6 and 3.7 to travis config?